texar/torch/modules/connectors/connectors.py

```diff
@@ -516,7 +501,7 @@ class ReparameterizedStochasticConnector(ConnectorBase):
 
     def __init__(self,
                  output_size: OutputSize,
-                 mlp_input_size: Union[torch.Size, MaybeTuple[int], int],
+                 mlp_input_size: MaybeTuple[int],
```

Comment:
Do we still accept `torch.Size`?

Reply:
Yeah, `torch.Size` can pass `Tuple` too, so I did this change, I re-think about it that specifying it makes more sense.